crates/rethnet_evm_napi/src/context.rs

```diff
@@ -0,0 +1,79 @@
+use std::{io, sync::Arc};
+
+use napi::{
+    tokio::runtime::{Builder, Runtime},
+    Status,
+};
+use napi_derive::napi;
+use tracing_subscriber::{prelude::*, EnvFilter, Registry};
+
+#[napi]
+#[derive(Debug)]
+pub struct RethnetContext {
+    inner: Arc<Context>,
+}
```

Comment:
What does NAPI-RS do with `Arc`s? I searched their docs but couldn't find it.

Reply:
Nothing. This is a native Rust object that is exposed to JS using a handle.